src/index.html

```diff
@@ -4,24 +4,407 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    />
+    <title>Moyo</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
```

Comment:
The stylesheet link is pointing to an SCSS file (`index.scss`). Ensure that your build process (e.g., using Parcel) is correctly set up to handle SCSS files, as browsers do not natively support SCSS.